support-frontend/assets/helpers/user/user.js

```diff
@@ -89,9 +89,9 @@ const init = (dispatch: Function, actions: UserSetStateActions = defaultUserActi
   const emailMatchesTestUser = (): boolean => {
     const testUsername = cookie.get('_test_username');
     if (emailFromBrowser && testUsername) {
-      return emailFromBrowser.toLowerCase().startsWith(testUsername.toLowerCase())
+      return emailFromBrowser.toLowerCase().startsWith(testUsername.toLowerCase());
```

Comment:
I saw these changes to this file on another PR  - I think it's the result of a `lint --fix`